src/main/rx/crsf.c

```diff
@@ -150,6 +183,25 @@ typedef struct crsfPayloadLinkstatistics_s {
     int8_t downlink_SNR;
 } crsfLinkStatistics_t;
 
+#if defined(USE_CRSF_V3)
+typedef struct crsfPayloadLinkstatisticsRx_s {
+    uint8_t downlink_RSSI_1;
+    uint8_t downlink_RSSI_1_percentage;
+    uint8_t downlink_Link_quality;
+    int8_t downlink_SNR;
+    uint8_t uplink_power;
+} crsfLinkStatisticsRx_t;
+
+typedef struct crsfPayloadLinkstatisticsTx_s {
+    uint8_t uplink_RSSI;
+    uint8_t uplink_RSSI_percentage;
+    uint8_t uplink_Link_quality;
+    int8_t uplink_SNR;
+    uint8_t downlink_power;
+    uint8_t uplink_FPS;
```

Comment:
No plans for links speeds >255hz?

Reply:
actually this variable holds the number of (actual fps / 10). so it will support at max 2550 hz. :)